packages/controls/src/color.ts

```diff
@@ -1,4 +1,3 @@
-import { match } from 'ts-pattern'
```

Comment:
Hmm, we probably want to set up ESLint to run on CI

Reply:
It does, that's how I caught this

Reply:
Awesome!